.docker/entrypoint.sh

```diff
@@ -4,7 +4,7 @@ start_server() {
 }
 
 if bundle check; then
```

Comment:
I remember you mentioned in slack that we weren't using this that you could tell. Does that continue to be the case?

Reply:
As far as I know! I had this a while ago to do fun things with `docker` but I think it got removed at some point. We can nix it if you'd like.